networkx/utils/backends.py

```diff
@@ -117,19 +117,35 @@ def _get_plugins(group, *, load_and_call=False):
                 rv[ep.name] = ep.load()()
             except Exception as exc:
                 warnings.warn(
-                    f"Error encountered when loading info for plugin {ep.name}: {exc}",
+                    f"Error encountered when loading info for backend {ep.name}: {exc}",
                     RuntimeWarning,
                     stacklevel=2,
                 )
         else:
             rv[ep.name] = ep
-    # nx-loopback plugin is only available when testing (added in conftest.py)
+    # nx-loopback backend is only available when testing (added in conftest.py)
     rv.pop("nx-loopback", None)
     return rv
 
 
-plugins = _get_plugins("networkx.plugins")
-plugin_info = _get_plugins("networkx.plugin_info", load_and_call=True)
+# Rename "plugin" to "backend", and give backends a release cycle to update.
+backends = _get_backends("networkx.plugins")
+backend_info = _get_backends("networkx.plugin_info", load_and_call=True)
+
+backends.update(_get_backends("networkx.backends"))
+backend_info.update(_get_backends("networkx.backend_info", load_and_call=True))
```

Comment:
I don't think we need to do this, but I guess it's better to do be conservative anyway :)